docs/settings.rst

```diff
@@ -128,11 +128,17 @@ Basic settings
 .. data:: LOG_FILTER = []
 
    A list of tuples containing the logging level (up to ``warning``) and the
-   message to be ignored.
+   message to be ignored or containing the type (either ``string`` or ``regex``)
+   , the logging level and a regular expression. If the latter format is used,
+   messages that match the regular expression will not be shown.
 
    Example::
 
-      LOG_FILTER = [(logging.WARN, 'TAG_SAVE_AS is set to False')]
+      LOG_FILTER = [
+        (logging.WARN, 'TAG_SAVE_AS is set to False'),
+        ('string', logging.WARN, 'Empty theme folder. Using `basic` theme.'),
```

Comment:
If I understand correctly, 2 item one the one starting with `"string"` are essentially same thing. And I'm guessing, `"string"` is added to make it consistent(?) with `"regex"`. This duplication is a bit redundant. I'd much rather have the "2 item" version deprecated but internally still supported by auto-changing it to "3 item string" version with a warning issued.